WhateverGreen/kern_igfx_backlight.cpp

```diff
@@ -357,6 +374,17 @@ void IGFX::BacklightSmoother::processKernel(KernelPatcher &patcher, DeviceInfo *
 		brightnessRange.second = UINT32_MAX;
 	}
 	
+	sines = static_cast<double*>(kern_os_malloc(steps * sizeof(double)));
```

Comment:
Use Buffer::create<double>, in my opinion. It is specifically for that purpose.

Reply:
Removed